### PR TITLE
Implement the Web Crypto API

### PR DESCRIPTION
**Pull request by timonson**
This PR implements deno's and the browser's new [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto).

- closes https://github.com/timonson/djwt/issues/47
- closes https://github.com/timonson/djwt/issues/48
- closes https://github.com/timonson/djwt/issues/51
- closes https://github.com/timonson/djwt/issues/53

### CODE REVIEW

**Comment by timonson** [comment]
I would appreciate a code review a lot.

The algorithm `ECDSA` is still missing but will be implemented next.

File: README.md [hunk 1]
```diff
@@ -6,43 +6,43 @@ Create and verify JSON Web Tokens with deno.
...
-Takes a `header`, `payload` and `key` and returns the url-safe encoded `jwt`.
+Takes `header`, `payload` and `key` and returns the url-safe encoded `jwt`.
 
 ```typescript
-import { create } from "https://deno.land/x/djwt@$VERSION/mod.ts"
-
-const jwt = await create({ alg: "HS512", typ: "JWT" }, { foo: "bar" }, "secret")
-// eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJmb28iOiJiYXIifQ.WePl7achkd0oGNB8XRF_LJwxlyiPZqpdNgdKpDboAjSTsWq-aOGNynTp8TOv8KjonFym8vwFwppXOLoLXbkIaQ
+import { create } from "https://deno.land/x/djwt@$VERSION/mod.ts";
+const key = await crypto.subtle.generateKey(
```
Comment (iugo) [inline README.md:13]: `importKey` method should also be added to README.

Comment (timonson) [reply README.md:13]: Hi thanks for taking the time. I will add `importKey` but `generateKey` has already been implemented and it works: https://github.com/denoland/deno/blob/main/ext/crypto/00_crypto.js

The reason I am wary about using `importKey` is that people might use too short or not recommended key lengths.

Comment (iugo) [reply README.md:13]: Sorry, I was wrong just now. deno has not yet implemented `exportKey`.

**Pull request merged**